src/renderer_vk.cpp

```diff
@@ -1570,6 +1570,7 @@ VK_IMPORT_INSTANCE
 				m_deviceFeatures.shaderClipDistance   = supportedFeatures.shaderClipDistance;
 				m_deviceFeatures.shaderCullDistance   = supportedFeatures.shaderCullDistance;
 				m_deviceFeatures.shaderResourceMinLod = supportedFeatures.shaderResourceMinLod;
+				m_deviceFeatures.geometryShader = supportedFeatures.geometryShader;
```

Comment:
Ideally this would check for the requested capabilities, since enabling device features can incur a performance cost. This is done for e.g. `.imageCubeArray` up top as well.